compiler/tests/asm.rs

```diff
@@ -88,14 +95,16 @@ fn block_to_block() {
     gen_halo2_proof(f, slice_to_vec(&i));
 }
 
-#[test]
-#[ignore = "Too slow"]
-fn keccak() {
-    let f = "keccak.asm";
-    let i = [];
-    verify_asm::<GoldilocksField>(f, slice_to_vec(&i));
-    gen_halo2_proof(f, slice_to_vec(&i));
-}
+// Commented out until thibaut provides the "keccak.asm"
```

Comment:
I think this can be removed! `keccak.asm` used to be checked-in for the benchmark, but now that the benchmark computes it from scratch during setup, this file doesn't exist anymore.